src/modals/reveal-seed-phrase-modal/reveal-seed-phrase-form-content/reveal-seed-phrase-form-content.tsx

```diff
@@ -49,7 +49,10 @@ export const RevealSeedPhraseFormContent: FC<FormikProps<RevealSeedPhraseModalFo
       />
       <RevealSeedPhraseView publicKeyHash={values.account.publicKeyHash} />
       <Divider size={formatSize(16)} />
-      <RevealAttention />
+      <Disclaimer
+        title="Attention!"
+        texts={['DO NOT share this set of chars with anyone!', 'It can be used to steal your current account.']}
```

Comment:
Maybe better use constants for static text somewhere in one place and use it everywhere?

Reply:
there are different texts in different places of app